src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_RANDOM_NUMBER = 100;
+    private final Random random;
+    
+    public Lottery() {
+        this.random = new Random();
+    }
+
+    public Ball getRandomBall() {
+        int randomNumber = random.nextInt(MAX_RANDOM_NUMBER);
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/156338484/208949ad-cfca-44ee-9a2f-c82cbf0d7f36)

Reply:
I made them local, but the previous mentor asked me to make them class-level.

Reply:
You missed ColorSupplier with randomNumber ))
randomNumber must be local